### DISCUSSION TITLE
failed get link: invalid X-Device-Id

### DISCUSSION THREAD

**Discussion by wshaow** [AlistGo/alist#3378] [General]
使用阿里云盘出现这个问题请问怎么解决
failed get link: invalid X-Device-Id

**Comment by mugqsir** [comment]
同样的问题，昨前天还好好的，今天就这样了，是什么原因呀，大佬们能解决下吗，谢谢！

**Reply by SHutdog** [reply]
我也是一样

**Reply by kingfanslwh** [reply]
我也是一样，但是挂载别人的没问题

**Comment by SHutdog** [comment]
invalid X-Device-Id

**Comment by fffy520** [comment]
failed get link: UserDeviceIllegality:invalid X-Device-Id
看起来好像是被封了~我今天也出现这个，昨天都正常的

**Comment by luoxianchun** [comment]
请问smb和webdav挂载后图片无法预览是什么问题？

**Comment by testname000003** [comment]
我也是这样，之前一直正常，其他存储正常，阿里云盘就这个提示，能刷新目录出来，文件无法下载，更新了token也是这样

**Comment by wshaow** [comment]
应该是被封了

**Comment by billzhen870** [comment]
我也是一样的

**Comment by clsoho** [comment]
{"code":500,"message":"failed get link: invalid X-Device-Id","data":null}
这是被封号了吗？在手机端播放正常呢

**Comment by pinlu1986** [comment]
今天开始不行了,更新了token也是一样

**Comment by Cue-vi** [comment]
同样出现了问题failed get link: invalid X-Device-Id

**Comment by Bobxyo** [comment]
原来不是我一个人这样……坐等大佬解决吧

**Comment by peterwan1989** [comment]
一样的问题，坐等大佬解决，手机app和PC端软件里都可以访问打开文件，挂载软路由alist后无法访问文件，但是可以打开目录，打开文件就显示：failed get link: invalid X-Device-Id

**Comment by barry820820** [comment]
一样的问题

**Comment by yeenjie123456** [comment]
我也一样的问题，昨天还好好的😅今天就不行了

**Comment by NG1988** [comment]
更新最新版本的alist之后可以恢复正常使用

**Reply by mavericklou** [reply]
已解决

**Reply by crosslife** [reply]
请问是哪个版本，3.11吗？

**Reply by Huanxige52** [reply]
哪个版本解决了？

**Reply by NG1988** [reply]
就是3.11  但是昨天更新后还是正常的，今日开始又有点抽风了

**Reply by Huanxige52** [reply]
我今天刚更新的 感觉还行 就是下载的时候跳转太慢了 很慢 但是可以正常用

**Comment by barry820820** [comment]
是的

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: 404NotFound ***@***.***>
发送时间: Thursday, February 16, 2023 11:14:09 AM
收件人: alist-org/alist ***@***.***>
抄送: barry820820 ***@***.***>; Comment ***@***.***>
主题: Re: [alist-org/alist] failed get link: invalid X-Device-Id (Discussion #3378)


请问是哪个版本，3.11吗？

―

**Comment by chengirii** [comment]
> 使用阿里云盘出现这个问题请问怎么解决 failed get link: invalid X-Device-Id

更新到最新版本 [3.11.0] 就恢复正常了

**Comment by barry820820** [comment]
11.3

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: 欢喜哥 ***@***.***>
发送时间: Thursday, February 16, 2023 1:22:48 PM
收件人: alist-org/alist ***@***.***>
抄送: barry820820 ***@***.***>; Comment ***@***.***>
主题: Re: [alist-org/alist] failed get link: invalid X-Device-Id (Discussion #3378)


哪个版本解决了？

―

**Comment by Huanxige52** [comment]
我今天刚更新的3.11 目前还没感觉出来啥大问题 就是跳转的时候太慢了 等半天




高兵
***@***.***



&nbsp;





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "alist-org/alist"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月16日(星期四) 下午2:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [alist-org/alist] failed get link: invalid X-Device-Id (Discussion #3378)





 
就是3.11  但是昨天更新后还是正常的，今日开始又有点抽风了

**Comment by evanmyt** [comment]
failed get objs: failed to list objs: invalid X-Device-Id

**Comment by balabalazhoucj** [comment]
3.12.0问题依旧

**Comment by yikexiaoyao** [comment]
我是nas上面挂的，试了好几次不行，有人解决了吗

**Comment by Lieber2021** [comment]
同样的提示，不知道咋处理
